django-stubs/views/generic/detail.pyi

```diff
@@ -1,20 +1,22 @@
-from typing import Any, Optional, Type
+from typing import Any, Optional, Type, TypeVar
 
 from django.views.generic.base import ContextMixin, TemplateResponseMixin, View
 
 from django.db import models
 from django.http import HttpRequest, HttpResponse
 
+T = TypeVar('T', bound=models.Model)
+
 class SingleObjectMixin(ContextMixin):
```

Comment:
Don't forget to use `Generic[T]` superclass!

Reply:
I might be out of my depth here - what do you mean? 

Btw, if you think it would be easier, feel free to jump in and take over at any point to round off the PR 👏